app/forms/webauthn_verification_form.rb

```diff
@@ -5,14 +5,16 @@ class WebauthnVerificationForm
   include ActionView::Helpers::TranslationHelper
   include Rails.application.routes.url_helpers
 
+  validates :screen_lock_error,
+            absence: { message: proc { |object| object.send(:screen_lock_error_message) } }
   validates :challenge,
             :authenticator_data,
             :client_data_json,
             :signature,
             :webauthn_configuration,
-            presence: { message: proc { |object| object.instance_eval { generic_error_message } } }
+            presence: { message: proc { |object| object.send(:generic_error_message) } }
   validates :webauthn_error,
-            absence: { message: proc { |object| object.instance_eval { generic_error_message } } }
+            absence: { message: proc { |object| object.send(:generic_error_message) } }
```

Comment:
why the switch to `#send`? fewer characters?

Reply:
Yes 😅 The `instance_eval` exceeded the character limit.

I'm assuming there's not much meaningful difference?

Reply:
nope no difference